packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/categories/CreateCategoriesTests.java

```diff
@@ -100,7 +100,7 @@ public void testCreateCategoryWithoutName_andFail()
                 .withCoreAPI()
                 .usingCategory(rootCategory)
                 .createSingleCategory(aCategory);
-        restClient.assertStatusCodeIs(BAD_REQUEST).assertLastError().containsSummary("Category name must not be null or empty");
+        restClient.assertStatusCodeIs(BAD_REQUEST).assertLastError().containsSummary("Category name must not be null nor empty");
```

Comment:
I would say "Category name must not be null or empty" but I don't know which is correct grammar. If I was using "nor" I would say something like "Category name must not be null, nor must it be empty."

Reply:
Ok, I will fix that.